src/game_interpreter_battle.cpp

```diff
@@ -596,21 +597,143 @@ bool Game_Interpreter_Battle::CommandManiacControlBattle(lcf::rpg::EventCommand
 	return true;
 }
 
-bool Game_Interpreter_Battle::CommandManiacControlAtbGauge(lcf::rpg::EventCommand const&) {
+bool Game_Interpreter_Battle::CommandManiacControlAtbGauge(lcf::rpg::EventCommand const& com) {
 	if (!Player::IsPatchManiac()) {
 		return true;
 	}
 
-	Output::Warning("Maniac Patch: Command ControlAtbGauge not supported");
+	// 0 actor
```

Comment:
Can you move all the comments after the ``case`` labels? E.g. as done here: https://github.com/EasyRPG/Player/blob/master/src/game_interpreter_control_variables.cpp#L70

(okay in the linked code they are the line after and not on the same line but doesn't matter where you locate them)

Reply:
Done! 

Although, I did like the fact that I could read them anywhere from the intellisense before: 
![image](https://github.com/user-attachments/assets/e5b0a433-f65c-4e8c-8955-ea42c97ba400)
Perhaps something to consider, but I'll adhere to the standard regardless :)

Reply:
best would be a local enum in the function and then using these enum-values for the cases (but I will not ask you to do this additional work for us)